app/components/subject-viewer.cjsx

```diff
@@ -201,6 +204,7 @@ module.exports = React.createClass
     @props.onFrameChange frame
 
   showMetadata: ->
+    @logSubjClick "metadata"
```

Comment:
This is throwing `TypeError: this.logSubjClick is not a function` when I tried clicking on the metadata button in the subject viewer.